phpunit/fonts-api/gutenbergRegisterFontsFromThemeJson-test.php

```diff
@@ -0,0 +1,360 @@
+<?php
+
+require_once __DIR__ . '/wp-fonts-testcase.php';
+
+/**
+ * Test gutenberg_register_fonts_from_theme_json().
+ *
+ * @package WordPress
+ * @subpackage Fonts API
+ *
+ * @since X.X.X
```

Comment:
Should this be more specific?
```suggestion
 * @since 15.7.0
```

Reply:
The `X.X.X` is a placeholder for when it's introduced into WP Core. The plan is to introduce the function and this test class into Core, hopefully in 6.3. So the `X.X.X`s in the API would all be updated to `6.3.0` at that time.

Reply:
Thank you for the explanation.
I have not encountered this notation before.
Now I know that such a practice exists.